zerver/views/streams.py

```diff
@@ -170,11 +172,12 @@ def list_subscriptions_backend(request, user_profile):
 FuncKwargPair = Tuple[Callable[..., HttpResponse], Dict[str, Union[int, Iterable[Any]]]]
 
 @has_request_variables
-def update_subscriptions_backend(request, user_profile,
-                                 delete=REQ(validator=check_list(check_string), default=[]),
-                                 add=REQ(validator=check_list(check_dict([('name', check_string)])),
-                                         default=[])):
-    # type: (HttpRequest, UserProfile, Iterable[Text], Iterable[Mapping[str, Any]]) -> HttpResponse
+def update_subscriptions_backend(
+        request: HttpRequest, user_profile: UserProfile,
+        delete: Iterable[Text]=REQ(validator=check_list(check_string), default=[]),
+        add: Iterable[Mapping[str, Any]]=REQ(validator=check_list(check_dict([('name', check_string)])),
+                                             default=[]),
```

Comment:
I found the `default` argument here a little hard to visually match with the other argument (`validator`) or the function it was passed to, so I reflowed this line while merging.